esphome/components/homeassistant/__init__.py

```diff
@@ -13,6 +13,13 @@
     }
 )
 
+HOME_ASSISTANT_IMPORT_CONTROL_SCHEMA = cv.Schema(
+    {
+        cv.Required(CONF_ENTITY_ID): cv.entity_id,
```

Comment:
To make sure everyone understand directly that the entity_id is related to home-assistant would it not be to change:  
```suggestion
          cv.Required(CONF_HOMEASSISTANT_ENTITY): cv.entity_id
```